ddtrace/llmobs/_llmobs.py

```diff
@@ -226,6 +227,21 @@ def disable(cls) -> None:
 
         log.debug("%s disabled", cls.__name__)
 
+    @classmethod
+    def annotation_context(cls, tags: Optional[Dict[str, Any]] = None) -> AnnotationContext:
+        """
+        Sets specified attributes on all spans created while the returned AnnotationContext is active.
+
+        :param tags: Dictionary of JSON serializable key-value tag pairs to set or update on the LLMObs span
+                     regarding the span's context.
+        """
+
+        def mini_annotate(span):
+            if tags:
+                cls._tag_span_tags(span, tags)
```

Comment:
Why not just use `LLMObs.annotate(span, tags=tags)`?

Reply:
`annotate` requires the `SPAN_KIND` tag to be set, which isn't available on span start (since tags are set after spans are started)

Reply:
I think I'd prefer to just move up the `metadata/tags/metrics` tagging in the annotate() function to be before the span kind check, since the span kind check is only necessary for I/O

Reply:
That makes sense!